monodoc/Test/Monodoc/HelpSourceTests.cs

```diff
@@ -108,7 +109,8 @@ IEnumerable<Node> GetLeaves (Node node)
 		[Test]
 		public void ReachabilityWithShortGenericNotationTest ()
 		{
-			var rootTree = RootTree.LoadTree (Path.GetFullPath (BaseDir), false);
+			string path = Path.GetFullPath(Path.Combine(Path.GetDirectoryName(this.GetType().Assembly.Location), BaseDir));
```

Comment:
Same issue as above, and given that you now have duplicated path lookup code, perhaps a `static` helper method would be better?

    static RootTree LoadTree ()
    {
        var path = Path.GetFullPath (
                Path.Combine (
                    Path.GetDirectoryName (this.GetType ().Assembly.Location),
                    BaseDir));
        return RootTree.LoadTree (path, false);
    }